src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -187,38 +194,142 @@ public static void SetParameterValue(Autodesk.Revit.DB.Parameter param, bool val
         public static double GetConvertedParameterValue(Autodesk.Revit.DB.Parameter param, double value)
         {
             var paramType = param.Definition.ParameterType;
```

Comment:
Is it better to get the UnitType directly here, and eliminate the need for a dictionary?

Like this: `unitType = param.Definition.UnitType`